roachtest: activerecord failed

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2574821&tab=buildLog) on [master@63fba7f1ab321088619d04aa9a44e61d018016f6](https://github.com/cockroachdb/cockroach/commits/63fba7f1ab321088619d04aa9a44e61d018016f6):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:760: 
		Tests run on Cockroach v21.1.0-alpha.1-946-g63fba7f1ab
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2574821&tab=artifacts#/activerecord)
Related:
- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2582294&tab=buildLog) on [master@f9b44f59d0451dc9efbb771baac3e88540485629](https://github.com/cockroachdb/cockroach/commits/f9b44f59d0451dc9efbb771baac3e88540485629):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:760: 
		Tests run on Cockroach v21.1.0-alpha.1-1027-gf9b44f59d0
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2582294&tab=artifacts#/activerecord)
Related:
- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2590485&tab=buildLog) on [master@7b0ccdda99b81613e70f421c9374483c3feddff3](https://github.com/cockroachdb/cockroach/commits/7b0ccdda99b81613e70f421c9374483c3feddff3):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:760: 
		Tests run on Cockroach v21.1.0-alpha.1-1120-g7b0ccdda99
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2590485&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2597708&tab=buildLog) on [master@3d01fda046c609ee5947e2f41c099dae35b206c1](https://github.com/cockroachdb/cockroach/commits/3d01fda046c609ee5947e2f41c099dae35b206c1):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:760: 
		Tests run on Cockroach v21.1.0-alpha.1-1177-g3d01fda046
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2597708&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2615395&tab=buildLog) on [master@d86781c07065421f4a4d8bf5d988900ab07fdce5](https://github.com/cockroachdb/cockroach/commits/d86781c07065421f4a4d8bf5d988900ab07fdce5):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.1-1339-gd86781c070
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6090 tests passed
		3 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2615395&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2622646&tab=buildLog) on [master@1a459a81dba35b6a091f0a2954aa33d50f1e5d24](https://github.com/cockroachdb/cockroach/commits/1a459a81dba35b6a091f0a2954aa33d50f1e5d24):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.1-1428-g1a459a81db
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2622646&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2629400&tab=buildLog) on [master@171cc86d6d84c110197485b5d886d77cddc12399](https://github.com/cockroachdb/cockroach/commits/171cc86d6d84c110197485b5d886d77cddc12399):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.1-1508-g171cc86d6d
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2629400&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2642936&tab=buildLog) on [master@100c09f4f6eb3f5b18a67ec4bbfdfe989e0d6ce2](https://github.com/cockroachdb/cockroach/commits/100c09f4f6eb3f5b18a67ec4bbfdfe989e0d6ce2):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.1-1668-g100c09f4f6-dirty
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2642936&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2651620&tab=buildLog) on [master@0e98670c8fcec566937e899fdf77d2a68c702d62](https://github.com/cockroachdb/cockroach/commits/0e98670c8fcec566937e899fdf77d2a68c702d62):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.1-1744-g0e98670c8f
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2651620&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2653401&tab=buildLog) on [master@5b33e6dfc47000de831745a851e3bf9e2cf7fd95](https://github.com/cockroachdb/cockroach/commits/5b33e6dfc47000de831745a851e3bf9e2cf7fd95):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.1-1748-g5b33e6dfc4
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6087 tests passed
		6 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		5 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_take_an_array_and_assign_the_attributes_to_the_associated_models (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_work_with_update_as_well (unexpected)
		--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_take_a_hash_with_string_keys_and_assign_the_attributes_to_the_associated_models (unexpected)
		--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_not_load_association_when_updating_existing_records (unexpected)
		--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_also_work_with_a_HashWithIndifferentAccess (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2653401&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2657140&tab=buildLog) on [master@7853fd32de8b6dea869f2a2a92dcd7506f4a8998](https://github.com/cockroachdb/cockroach/commits/7853fd32de8b6dea869f2a2a92dcd7506f4a8998):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.3-213-g7853fd32de
		Tests run against activerecord 6.0.3.4
		6093 Total Tests Run
		6091 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2657140&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2665935&tab=buildLog) on [master@57de93a997db31d97cd7e23fb866fe0e44f90de2](https://github.com/cockroachdb/cockroach/commits/57de93a997db31d97cd7e23fb866fe0e44f90de2):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.3-296-g57de93a997-dirty
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6092 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2665935&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2670113&tab=buildLog) on [master@e9e372122a2e3db7090b5705da07128f828e2441](https://github.com/cockroachdb/cockroach/commits/e9e372122a2e3db7090b5705da07128f828e2441):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.3-343-ge9e372122a
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6091 tests passed
		3 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2670113&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2674690&tab=buildLog) on [master@9969862335cf501c1aa67a4aaf018ad5dc8e85e8](https://github.com/cockroachdb/cockroach/commits/9969862335cf501c1aa67a4aaf018ad5dc8e85e8):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.3-377-g9969862335-dirty
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6092 tests passed
		2 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2674690&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2676655&tab=buildLog) on [master@5971ecb9dd1a25c81cd6012d6be1ff922802eae5](https://github.com/cockroachdb/cockroach/commits/5971ecb9dd1a25c81cd6012d6be1ff922802eae5):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.3-393-g5971ecb9dd-dirty
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6091 tests passed
		3 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_from (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2676655&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2688408&tab=buildLog) on [master@3c223f5f5162103110a790743b687ef2bf952489](https://github.com/cockroachdb/cockroach/commits/3c223f5f5162103110a790743b687ef2bf952489):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:228,orm_helpers.go:154,activerecord.go:224,activerecord.go:236,test_runner.go:767: 
		Tests run on Cockroach v21.1.0-alpha.3-549-g3c223f5f51
		Tests run against activerecord 6.0.3.4
		6094 Total Tests Run
		6091 tests passed
		3 tests failed
		56 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlMoneyTest#test_money_regex_backtracking (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2688408&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2693234&tab=buildLog) on [master@83e70ce84b740e27e721c3b73c38a4b8b515094a](https://github.com/cockroachdb/cockroach/commits/83e70ce84b740e27e721c3b73c38a4b8b515094a):

```
		--- FAIL: BasicsTest#test_equality_of_relation_and_array (unexpected)
		--- FAIL: CockroachDB::WithAnnotationsTest#test_belongs_to_with_annotation_includes_a_query_comment (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_integer_for_binary_column (unexpected)
		--- FAIL: BasicsTest#test_invalid_limit (unexpected)
		--- FAIL: BasicsTest#test_toggle_attribute (unexpected)
		--- FAIL: BasicsTest#test_clone_of_new_object_marks_attributes_as_dirty (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_custom_join_table (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_empty_hash_and_no_foreign_key (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_copies_bind_params_in_the_right_order (unexpected)
		--- FAIL: BasicsTest#test_incomplete_schema_loading (unexpected)
		--- FAIL: BasicsTest#test_create_after_initialize_with_array_param (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_not_polymorphic_id_and_type_as_nor_is_deprecated (unexpected)
		--- FAIL: BasicsTest#test_table_name_with_2_abstract_subclasses (unexpected)
		--- FAIL: BasicsTest#test_selecting_a_record_does_not_raise_if_preventing_writes (unexpected)
		--- FAIL: BasicsTest#test_marshal_inspected_round_trip (unexpected)
		--- FAIL: BasicsTest#test_default (unexpected)
		--- FAIL: CockroachDB::WithAnnotationsTest#test_has_many_through_with_annotation_includes_a_query_comment_when_eager_loading (unexpected)
		--- FAIL: BasicsTest#test_equality_of_destroyed_records (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_multiple_stis_and_order (unexpected)
		--- FAIL: BasicsTest#test_switching_between_table_name (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_on_association_with_select_relation (unexpected)
		--- FAIL: BasicsTest#test_updating_a_record_raises_if_preventing_writes (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_two_levels_with_condition (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_create (unexpected)
		--- FAIL: CockroachDB::WithAnnotationsTest#test_has_many_through_with_annotation_includes_a_query_comment (unexpected)
		--- FAIL: FoxyFixturesTest#test_ignores_belongs_to_symbols_if_association_and_foreign_key_are_named_the_same (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_include_checks_if_record_exists_if_target_not_loaded (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_two_levels_with_two_has_many_associations (unexpected)
		--- FAIL: BasicsTest#test_changing_a_connection_handler_in_a_main_thread_does_not_poison_the_other_threads (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_proper_usage_of_primary_keys_and_join_table (unexpected)
		--- FAIL: BasicsTest#test_column_types_typecast (unexpected)
		--- FAIL: FoxyFixturesTest#test_supports_label_interpolation_for_integer_label (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_update_columns_after_push_without_duplicate_join_table_rows (unexpected)
		--- FAIL: BasicsTest#test_preventing_writes_with_multiple_handlers (unexpected)
		--- FAIL: BasicsTest#test_limit_should_take_value_from_latest_limit (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_join_table_alias (unexpected)
		--- FAIL: BasicsTest#test_clone_preserves_subtype (unexpected)
		--- FAIL: BasicsTest#test_bignum_pk (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_dynamic_find_should_respect_association_order (unexpected)
		--- FAIL: BasicsTest#test_comparison_with_different_objects (unexpected)
		--- FAIL: BasicsTest#test_attribute_names (unexpected)
		--- FAIL: BasicsTest#test_column_name_properly_quoted (unexpected)
		--- FAIL: BasicsTest#test_records_of_different_classes_have_different_hashes (unexpected)
		--- FAIL: BasicsTest#test_table_exists (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_dynamic_find_all_should_respect_readonly_access (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_store_association_with_a_polymorphic_relationship (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_associations_with_conditions (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2693234&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2699361&tab=buildLog) on [master@64c4aef909f4382523cd9248341ca9f4448d841a](https://github.com/cockroachdb/cockroach/commits/64c4aef909f4382523cd9248341ca9f4448d841a):

```
		--- FAIL: HasManyAssociationsTest#test_create_followed_by_save_does_not_load_target (unexpected)
		--- FAIL: ReflectionTest#test_columns_are_returned_in_the_order_they_were_declared (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_delete_all_on_association_clears_scope (unexpected)
		--- FAIL: CockroachDB::WithAnnotationsTest#test_belongs_to_with_annotation_includes_a_query_comment (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_association_proxy_transaction_method_starts_transaction_in_association_class (unexpected)
		--- FAIL: PostgresqlBitStringTest#test_roundtrip (unexpected)
		--- FAIL: ReflectionTest#test_has_and_belongs_to_many_reflection (unexpected)
		--- FAIL: InheritanceTest#test_inheritance_find_all (unexpected)
		--- FAIL: FinderTest#test_find_by_with_multi-arg_conditions_returns_the_first_matching_record (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache_with_dependent_delete_all (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_ordered_has_many_through (unexpected)
		--- FAIL: PersistenceTest#test_create_many (unexpected)
		--- FAIL: FinderTest#test_select_value (unexpected)
		--- FAIL: CalculationsTest#test_should_return_integer_average_if_db_returns_such (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_none_should_return_false_if_any (unexpected)
		--- FAIL: BasicsTest#test_attributes_on_dummy_time (unexpected)
		--- FAIL: BasicsTest#test_non_valid_identifier_column_name (unexpected)
		--- FAIL: FinderTest#test_find_with_large_number (unexpected)
		--- FAIL: ReflectionTest#test_always_validate_association_if_explicit (unexpected)
		--- FAIL: HasManyAssociationsTest#test_first_or_create_adds_the_record_to_the_association (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_saving_multiple_relationships (unexpected)
		--- FAIL: UniquenessValidationTest#test_validate_uniqueness_of_custom_primary_key (unexpected)
		--- FAIL: CalculationsTest#test_should_count_field_in_joined_table (unexpected)
		--- FAIL: CalculationsTest#test_pluck_with_multiple_columns_and_includes (unexpected)
		--- FAIL: AssociationCallbacksTest#test_has_and_belongs_to_many_after_add_called_after_save (unexpected)
		--- FAIL: RelationTest#test_preloading_with_associations_and_merges (unexpected)
		--- FAIL: ReflectionTest#test_includes_accepts_symbols (unexpected)
		--- FAIL: EagerAssociationTest#test_including_duplicate_objects_from_belongs_to (unexpected)
		--- FAIL: AssociationCallbacksTest#test_has_and_belongs_to_many_add_callback (unexpected)
		--- FAIL: StoreTest#test_reading_store_attributes_through_accessors (unexpected)
		--- FAIL: AssociationsTest#test_using_limitable_reflections_helper (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_associations_on_new_records_use_null_relations (unexpected)
		--- FAIL: AttributeMethodsTest#test_update_array_content (unexpected)
		--- FAIL: BasicsTest#test_reload (unexpected)
		--- FAIL: FinderTest#test_first_have_determined_order_by_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnsTest#test_add_rename (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_update_ids_with_conditions (unexpected)
		--- FAIL: CalculationsTest#test_pluck_with_reserved_words (unexpected)
		--- FAIL: PessimisticLockingTest#test_lock_raises_when_the_record_is_dirty (unexpected)
		--- FAIL: FinderTest#test_find_by_empty_ids (unexpected)
		--- FAIL: TransactionTest#test_manually_rolling_back_a_transaction (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_create_should_not_raise_exception_when_join_record_has_errors (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_none_should_defer_to_collection_if_using_a_block (unexpected)
		--- FAIL: ReflectionTest#test_read_attribute_names (unexpected)
		--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_is_only_there_when_versioning_is_on (unexpected)
		--- FAIL: BasicsTest#test_slice_accepts_array_argument (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_associate_existing_record_twice_should_add_records_twice (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2699361&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2702231&tab=buildLog) on [master@bf9744bad5a416a4b06907f0f3dd42896f7342f3](https://github.com/cockroachdb/cockroach/commits/bf9744bad5a416a4b06907f0f3dd42896f7342f3):

```
		--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache (unexpected)
		--- FAIL: CalculationsTest#test_pluck_with_selection_clause (unexpected)
		--- FAIL: RelationTest#test_create_or_find_by (unexpected)
		--- FAIL: EagerAssociationTest#test_should_work_inverse_of_with_eager_load (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_counter_cache (unexpected)
		--- FAIL: HasManyAssociationsTest#test_build_via_block (unexpected)
		--- FAIL: HasOneAssociationsTest#test_finding_using_primary_key (unexpected)
		--- FAIL: EagerAssociationTest#test_finding_with_includes_on_null_belongs_to_polymorphic_association (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_on_association_clears_scope (unexpected)
		--- FAIL: IntegrationTest#test_cache_key_for_newer_updated_on (unexpected)
		--- FAIL: HasManyAssociationsTest#test_transactions_when_replacing_on_new_record (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_habtm_in_multiple_queries_with_more_ids_than_database_can_handle (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_many_should_defer_to_collection_if_using_a_block (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all (unexpected)
		--- FAIL: CockroachDB::FixturesTest#test_multiple_clean_fixtures (unexpected)
		--- FAIL: RelationTest#test_using_a_custom_table_with_joins_affects_the_joins (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_reassignment_of_associated_id_updates_the_object (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_with_primary_key_updates_foreign_id_field_for_new_and_saved_records (unexpected)
		--- FAIL: BasicsTest#test_primary_key_and_references_columns_should_be_identical_type (unexpected)
		--- FAIL: RelationTest#test_multiple_selects (unexpected)
		--- FAIL: ReflectionTest#test_includes_accepts_strings (unexpected)
		--- FAIL: RelationTest#test_respond_to_dynamic_finders (unexpected)
		--- FAIL: BasicsTest#test_attributes_on_dummy_time_with_invalid_time (unexpected)
		--- FAIL: HasManyAssociationsTest#test_finding_with_condition_hash (unexpected)
		--- FAIL: HasManyAssociationsTest#test_unscopes_the_default_scope_of_associated_model_when_used_with_include (unexpected)
		--- FAIL: HasManyAssociationsTest#test_select_query_method (unexpected)
		--- FAIL: CalculationsTest#test_count_with_eager_loading_and_custom_select_and_order (unexpected)
		--- FAIL: ReflectionTest#test_integer_columns (unexpected)
		--- FAIL: PersistenceTest#test_update_columns_changing_id (unexpected)
		--- FAIL: CalculationsTest#test_should_get_maximum_of_field (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_explicit_join (unexpected)
		--- FAIL: HasManyAssociationsTest#test_associations_replace_in_memory_when_records_have_the_same_id (unexpected)
		--- FAIL: CalculationsTest#test_pick_two (unexpected)
		--- FAIL: BasicsTest#test_unicode_column_name (unexpected)
		--- FAIL: CalculationsTest#test_should_average_arel_attribute (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_destroying (unexpected)
		--- FAIL: CalculationsTest#test_should_count_with_manual_distinct_select_and_distinct (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_load_has_many_through_with_string_keys (unexpected)
		--- FAIL: EagerAssociationTest#test_finding_with_includes_on_has_one_association_with_same_include_includes_only_once (unexpected)
		--- FAIL: RelationTest#test_#where_with_set (unexpected)
		--- FAIL: BasicsTest#test_marshalling_with_associations (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_with_scope_with_joins (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_destroy_with_destroyed_parent (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_symbol_join_table (unexpected)
		--- FAIL: BasicsTest#test_deleting_a_record_raises_if_preventing_writes (unexpected)
		--- FAIL: HasManyAssociationsTest#test_modifying_a_through_a_has_many_should_raise (unexpected)
		--- FAIL: BasicsTest#test_primary_key_with_no_id (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2702231&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2707822&tab=buildLog) on [master@5cfd7e5553a3072a1490d392390dddf968844215](https://github.com/cockroachdb/cockroach/commits/5cfd7e5553a3072a1490d392390dddf968844215):

```
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_association_with_validate_false_does_not_run_associated_validation_callbacks_on_update (unexpected)
		--- FAIL: FinderTest#test_bind_variables_with_quotes (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_get_has_many_through_belongs_to_ids_with_conditions (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_polymorphic_nested_where (unexpected)
		--- FAIL: BasicsTest#test_find_keeps_multiple_group_values (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_dynamic_find_should_respect_association_include (unexpected)
		--- FAIL: InheritanceTest#test_alt_complex_inheritance (unexpected)
		--- FAIL: FinderTest#test_model_class_responds_to_second_bang (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_three_mappings_array (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_preloading_empty_through_with_polymorphic_source_association (unexpected)
		--- FAIL: FinderTest#test_model_class_responds_to_third_to_last_bang (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_find_with_merged_options (unexpected)
		--- FAIL: InheritanceTest#test_inheritance_save (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_empty_array_with_aggregate_having_multiple_mappings (unexpected)
		--- FAIL: BasicsTest#test_using_table_name_qualified_column_names_unless_having_SELECT_list_explicitly (unexpected)
		--- FAIL: InheritanceTest#test_different_namespace_subclass_should_load_correctly_with_store_full_sti_class_option (unexpected)
		--- FAIL: FinderTest#test_second_to_last_have_primary_key_order_by_default (unexpected)
		--- FAIL: BasicsTest#test_clone_preserves_subtype (unexpected)
		--- FAIL: BasicsTest#test_dup_with_aggregate_of_same_name_as_attribute (unexpected)
		--- FAIL: BasicsTest#test_records_of_different_classes_have_different_hashes (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_rewhere_on_root (unexpected)
		--- FAIL: BasicsTest#test_creating_a_record_raises_if_preventing_writes (unexpected)
		--- FAIL: BasicsTest#test_records_without_an_id_have_unique_hashes (unexpected)
		--- FAIL: CockroachDB::WithAnnotationsTest#test_has_many_through_with_annotation_includes_a_query_comment (unexpected)
		--- FAIL: FinderTest#test_exists_does_not_instantiate_records (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_copies_bind_params_in_the_right_order (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_polymorphic_sti_shallow_where (unexpected)
		--- FAIL: FinderTest#test_last_with_integer_and_reorder_should_use_sql_limit (unexpected)
		--- FAIL: BasicsTest#test_null_fields (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_has_one_reflection (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_associate_with_create (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_update_counter_caches_on_replace_association (unexpected)
		--- FAIL: BasicsTest#test_generated_relation_methods_module_name (unexpected)
		--- FAIL: FinderTest#test_second_with_offset (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_get_ids_for_has_many_through_with_conditions_should_not_preload (unexpected)
		--- FAIL: InheritanceTest#test_update_all_within_inheritance (unexpected)
		--- FAIL: FoxyFixturesTest#test_supports_sti (unexpected)
		--- FAIL: FinderTest#test_find_with_ids_with_limit_and_order_clause (unexpected)
		--- FAIL: BasicsTest#test_find_by_slug_with_array (unexpected)
		--- FAIL: FinderTest#test_find_by_on_attribute_that_is_a_reserved_word (unexpected)
		--- FAIL: BasicsTest#test_find_multiple_ordered_last (unexpected)
		--- FAIL: BasicsTest#test_find_by_slug_with_range (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_delete_through_belongs_to_with_dependent_nullify (unexpected)
		--- FAIL: BasicsTest#test_limit_without_comma (unexpected)
		--- FAIL: BasicsTest#test_invalid_limit (unexpected)
		--- FAIL: BasicsTest#test_default (unexpected)
		--- FAIL: InheritanceTest#test_new_with_unrelated_type (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2707822&tab=artifacts#/activerecord)
Related:
- #60650 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-60635](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-60635) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2712399&tab=buildLog) on [master@ec011620c7cf299fdbb898db692b36454defc4a2](https://github.com/cockroachdb/cockroach/commits/ec011620c7cf299fdbb898db692b36454defc4a2):

```
		--- FAIL: HasManyAssociationsTest#test_loading_association_in_validate_callback_doesnt_affect_persistence (unexpected)
		--- FAIL: FinderTest#test_find_with_string (unexpected)
		--- FAIL: FinderTest#test_second_to_last_have_primary_key_order_by_default (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_associate_new_by_building (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_one_through_join_model_with_conditions_on_the_through (unexpected)
		--- FAIL: RelationTest#test_find_all_with_multiple_should_use_and (unexpected)
		--- FAIL: HasManyAssociationsTest#test_prevent_double_firing_the_before_save_callback_of_new_object_when_the_parent_association_saved_in_the_callback (unexpected)
		--- FAIL: BasicsTest#test_when_#reload_called,_ignored_columns'_attribute_methods_are_not_defined (unexpected)
		--- FAIL: RelationTest#test_find_by!_requires_at_least_one_argument (unexpected)
		--- FAIL: DateTest#test_date_with_time_value (unexpected)
		--- FAIL: AttributeMethodsTest#test_attribute_method?_returns_false_if_the_table_does_not_exist (unexpected)
		--- FAIL: AttributeMethodsTest#test_setting_a_time_zone-aware_datetime_in_the_current_time_zone (unexpected)
		--- FAIL: FinderTest#test_find_by_two_attributes (unexpected)
		--- FAIL: FinderTest#test_find_with_ids_with_no_id_passed (unexpected)
		--- FAIL: CalculationsTest#test_count_with_eager_loading_and_custom_order_and_distinct (unexpected)
		--- FAIL: InheritanceTest#test_new_with_ar_base (unexpected)
		--- FAIL: AttributeMethodsTest#test_custom_field_attribute_predicate (unexpected)
		--- FAIL: FinderTest#test_count_by_sql (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_push_with_invalid_record (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_through_association_readonly_should_be_false (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_association_assignment_sticks (unexpected)
		--- FAIL: FinderTest#test_find_by_on_a_scope_does_not_perform_statement_caching (unexpected)
		--- FAIL: DirtyTest#test_datetime_attribute_can_be_updated_with_fractional_seconds (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_collection_delete_with_nonstandard_primary_key_on_belongs_to (unexpected)
		--- FAIL: BasicsTest#test_initialize_with_invalid_attribute (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_without_updated_at_attributes (unexpected)
		--- FAIL: RelationTest#test_select_with_subquery_in_from_does_not_use_original_table_name (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_new_aliased_to_build (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_order_string_with_unquoted_table_name (unexpected)
		--- FAIL: PersistenceTest#test_update_columns_changing_id (unexpected)
		--- FAIL: FinderTest#test_last_with_integer_and_order_should_use_sql_limit (unexpected)
		--- FAIL: InverseMultipleHasManyInversesForSameModel#test_that_we_can_create_associations_that_have_the_same_reciprocal_name_from_different_models (unexpected)
		--- FAIL: RelationTest#test_find_or_create_by_with_create_with (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_push_with_invalid_join_record (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_association_class (unexpected)
		--- FAIL: MultiParameterAttributeTest#test_multiparameter_attributes_on_date_with_all_empty (unexpected)
		--- FAIL: RelationTest#test_order_with_reorder_nil_removes_the_order (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_with_select (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependent_association_respects_optional_conditions_on_delete (unexpected)
		--- FAIL: IntegrationTest#test_cache_key_for_existing_record_is_not_timezone_dependent (unexpected)
		--- FAIL: TransactionTest#test_raising_exception_in_nested_transaction_restore_state_in_save (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_concat (unexpected)
		--- FAIL: InheritanceTest#test_where_create_bang_with_unrelated_type (unexpected)
		--- FAIL: RelationTest#test_order_using_scoping (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_proxy_should_not_respond_to_private_methods (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_with_nil (unexpected)
		--- FAIL: CounterCacheTest#test_reset_multiple_counters_with_touch:_true (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2712399&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2716822&tab=buildLog) on [master@c7e088826bc079620dfd3b5ae75d1c15cd9cd16d](https://github.com/cockroachdb/cockroach/commits/c7e088826bc079620dfd3b5ae75d1c15cd9cd16d):

```
		--- FAIL: TestDefaultAutosaveAssociationOnAHasManyAssociation#test_valid_adding_with_validate_false (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_with_bang_bubbles_errors_from_associations (unexpected)
		--- FAIL: FoxyFixturesTest#test_supports_inline_habtm (unexpected)
		--- FAIL: CalculationsTest#test_no_limit_no_offset (unexpected)
		--- FAIL: InheritanceTest#test_where_create_bang_with_subclass (unexpected)
		--- FAIL: AttributeMethodsTest#test_converted_values_are_returned_after_assignment (unexpected)
		--- FAIL: MultiParameterAttributeTest#test_create_with_multiparameter_attributes_setting_date_and_time_attribute (unexpected)
		--- FAIL: BasicsTest#test_utc_as_time_zone_and_new (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependence (unexpected)
		--- FAIL: BasicsTest#test_attributes_on_dummy_time (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_join_model_ignores_default_includes (unexpected)
		--- FAIL: CalculationsTest#test_#skip_query_cache!_for_a_grouped_calculation (unexpected)
		--- FAIL: MigrationTest#test_rename_table_with_prefix_and_suffix (unexpected)
		--- FAIL: OptimisticLockingTest#test_lock_destroy (unexpected)
		--- FAIL: CalculationsTest#test_should_order_by_grouped_field (unexpected)
		--- FAIL: CalculationsTest#test_should_return_integer_average_if_db_returns_such (unexpected)
		--- FAIL: AttributeMethodsTest#test_set_attributes_without_a_hash (unexpected)
		--- FAIL: AssociationCallbacksTest#test_has_and_belongs_to_many_after_add_called_after_save (unexpected)
		--- FAIL: MultiParameterAttributeTest#test_multiparameter_attributes_on_time_will_ignore_hour_if_missing (unexpected)
		--- FAIL: MultipleDbTest#test_transactions_across_databases (unexpected)
		--- FAIL: EagerAssociationTest#test_load_associated_records_in_one_query_when_a_few_ids_passed (unexpected)
		--- FAIL: MultiParameterAttributeTest#test_multiparameter_attributes_setting_date_attribute (unexpected)
		--- FAIL: DefaultScopingTest#test_default_scope_with_references_works_through_association (unexpected)
		--- FAIL: DirtyTest#test_previous_changes (unexpected)
		--- FAIL: TimestampTest#test_no_touching_threadsafe (unexpected)
		--- FAIL: DirtyTest#test_integer_zero_to_integer_zero_not_marked_as_changed (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: IntegrationTest#test_to_param_should_return_string (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_many_should_return_true_if_more_than_one (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_in_collection (unexpected)
		--- FAIL: QueryCacheTest#test_query_cache_dups_results_correctly (unexpected)
		--- FAIL: ConcurrentTransactionTest#test_assign_id_after_rollback (unexpected)
		--- FAIL: InheritanceTest#test_alt_complex_inheritance (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_all_sanitized (unexpected)
		--- FAIL: PolymorphicNonFullStiClassNamesTest#test_class_names_with_includes (unexpected)
		--- FAIL: FixturesWithoutInstantiationTest#test_accessor_methods (unexpected)
		--- FAIL: HasManyAssociationsTest#test_collection_size_after_building (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all_with_option_delete_all (unexpected)
		--- FAIL: AttributeMethodsTest#test_write_attribute_raises_ActiveModel::MissingAttributeError_when_the_attribute_does_not_exist (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_a_polymorphic_association_with_references_to_the_associated_table (unexpected)
		--- FAIL: HasManyAssociationsTest#test_first_or_create!_adds_the_record_to_the_association (unexpected)
		--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_update_children_when_autosave_is_true_and_parent_is_new_but_child_is_not (unexpected)
		--- FAIL: ConcurrentTransactionTest#test_read_attribute_after_rollback (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_cascaded_eager_association_loading_with_duplicated_includes (unexpected)
		--- FAIL: DefaultScopingTest#test_default_scope_as_block_referencing_scope (unexpected)
		--- FAIL: OptimisticLockingTest#test_lock_new_when_explicitly_passing_nil (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_through_empty_belongs_to (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2716822&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2721381&tab=buildLog) on [master@6601d827b814d4e85a1081b03bf2562d8ac2a4ab](https://github.com/cockroachdb/cockroach/commits/6601d827b814d4e85a1081b03bf2562d8ac2a4ab):

```
		--- FAIL: ConcurrentTransactionTest#test_manually_rolling_back_a_transaction (unexpected)
		--- FAIL: IntegrationTest#test_to_param_class_method_truncates_case_shown_in_doc (unexpected)
		--- FAIL: AttributeMethodsTest#test_attribute_present_with_booleans (unexpected)
		--- FAIL: HasOneAssociationsTest#test_dangerous_association_name_raises_ArgumentError (unexpected)
		--- FAIL: RelationTest#test_find_all_with_join (unexpected)
		--- FAIL: DirtyTest#test_attribute_before_last_save_returns_the_original_value_before_saving (unexpected)
		--- FAIL: CalculationsTest#test_sum_with_from_option (unexpected)
		--- FAIL: RelationTest#test_last (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_integer_for_string_column (unexpected)
		--- FAIL: ConcurrentTransactionTest#test_restore_active_record_state_for_all_records_in_a_transaction (unexpected)
		--- FAIL: CalculationsTest#test_should_reference_correct_aliases_while_joining_tables_of_has_many_through_association (unexpected)
		--- FAIL: BasicsTest#test_successful_comparison_of_like_class_records (unexpected)
		--- FAIL: HasManyAssociationsTest#test_building_the_associated_object_with_sti_subclass (unexpected)
		--- FAIL: FinderTest#test_find_by_one_missing_attribute (unexpected)
		--- FAIL: AttributeMethodsTest#test_attribute_writers_respect_access_control (unexpected)
		--- FAIL: AttributeMethodsTest#test_read_attribute_when_true (unexpected)
		--- FAIL: CalculationsTest#test_should_count_manual_select_with_group_with_count_all (unexpected)
		--- FAIL: EagerAssociationTest#test_has_and_belongs_to_many_should_not_instantiate_same_records_multiple_times (unexpected)
		--- FAIL: PersistenceTest#test_update_parameters (unexpected)
		--- FAIL: ModulesTest#test_eager_loading_in_modules (unexpected)
		--- FAIL: ReflectionTest#test_foreign_type (unexpected)
		--- FAIL: ConcurrentTransactionTest#test_rolling_back_in_a_callback_rollbacks_before_save (unexpected)
		--- FAIL: AttributeMethodsTest#test_generated_attribute_methods_ancestors_have_correct_module (unexpected)
		--- FAIL: AttributeMethodsTest#test_custom_field_attribute_predicate (unexpected)
		--- FAIL: HasOneAssociationsTest#test_create_when_parent_is_new_raises (unexpected)
		--- FAIL: BasicsTest#test_set_table_name_with_inheritance (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_with_array (unexpected)
		--- FAIL: HasOneAssociationsTest#test_has_one_with_touch_option_on_update (unexpected)
		--- FAIL: BasicsTest#test_table_name_guesses_with_inherited_prefixes_and_suffixes (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_associate_existing_record_twice_should_add_to_target_twice (unexpected)
		--- FAIL: CalculationsTest#test_pluck_with_reserved_words (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_out_of_range_value_assigning (unexpected)
		--- FAIL: BinaryTest#test_mixed_encoding (unexpected)
		--- FAIL: BasicsTest#test_current_scope_is_reset (unexpected)
		--- FAIL: FinderTest#test_exists_with_scope (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_deleting_all (unexpected)
		--- FAIL: RelationTest#test_includes_with_select (unexpected)
		--- FAIL: BasicsTest#test_unicode_column_name (unexpected)
		--- FAIL: CalculationsTest#test_should_count_manual_select_with_count_all (unexpected)
		--- FAIL: CalculationsTest#test_should_sum_field (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_loading_with_conditions_on_string_joined_table_preloads (unexpected)
		--- FAIL: EagerAssociationTest#test_including_duplicate_objects_from_has_many (unexpected)
		--- FAIL: BasicsTest#test_an_empty_transaction_does_not_raise_if_preventing_writes (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_ordered_has_many_through (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_duration_for_string_column (unexpected)
		--- FAIL: JsonAttributeTest#test_json_with_serialized_attributes (unexpected)
		--- FAIL: RelationTest#test_blank_like_arguments_to_query_methods_dont_raise_errors (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2721381&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2725104&tab=buildLog) on [master@9595a158f0233e1c3d86786ec4462dd39c7beb20](https://github.com/cockroachdb/cockroach/commits/9595a158f0233e1c3d86786ec4462dd39c7beb20):

```
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_with_includes_in_through_association_scope (unexpected)
		--- FAIL: HasManyAssociationsTest#test_defining_has_many_association_with_delete_all_dependency_lazily_evaluates_target_class (unexpected)
		--- FAIL: ReflectionTest#test_class_for_class_name (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_obeys_order_on_through_association (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_should_raise_exception_for_destroying_mismatching_records (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all_on_association_clears_scope (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_a_collection (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_pk_is_not_required_for_join (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_explicitly_joining_join_table (unexpected)
		--- FAIL: HasManyAssociationsTest#test_passes_custom_context_validation_to_validate_children (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_no_pk_join_table_append (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_size_on_an_association_that_has_not_been_loaded_performs_a_query (unexpected)
		--- FAIL: HasManyAssociationsTest#test_finding_with_condition_hash (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_get_ids_for_loaded_associations (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_create_bang_should_raise_exception_when_join_record_has_errors (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_first_after_reset (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dynamic_find_should_respect_association_order (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_ids (unexpected)
		--- FAIL: HasManyAssociationsTest#test_with_polymorphic_has_many_with_custom_columns_name (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_first_nth_or_last_on_existing_record_with_create_should_not_load_association (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many_with_belongs_to (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_distinct_after_the_fact (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_preload_multiple_instances_of_the_same_record (unexpected)
		--- FAIL: HasManyAssociationsTest#test_create_resets_cached_counters (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_none_should_defer_to_collection_if_using_a_block (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_proper_usage_of_primary_keys_and_join_table (unexpected)
		--- FAIL: HasManyAssociationsTest#test_replace_with_less_and_dependent_nullify (unexpected)
		--- FAIL: HasManyAssociationsTest#test_transactions_when_adding_to_persisted (unexpected)
		--- FAIL: HasManyAssociationsTest#test_restrict_with_error (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_symbol_join_table (unexpected)
		--- FAIL: HasManyAssociationsTest#test_get_ids_for_through (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_dynamic_find_all_should_respect_readonly_access (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_delete_all_on_association_clears_scope (unexpected)
		--- FAIL: ReflectionTest#test_type (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependence_on_account (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_respects_select (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_record_with_complex_joins (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_singleton_has_many_through (unexpected)
		--- FAIL: HasManyAssociationsTest#test_association_keys_bypass_attribute_protection (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_belongs_to_with_dependent_nullify (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_association_through_a_belongs_to_association_where_the_association_doesnt_exist (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_all_on_association_clears_scope (unexpected)
		--- FAIL: HasManyAssociationsTest#test_prevent_double_firing_the_before_save_callback_of_new_object_when_the_parent_association_saved_in_the_callback (unexpected)
		--- FAIL: HasManyAssociationsTest#test_create_children_could_be_rolled_back_by_after_save (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_collection_exists (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_one_should_count_instead_of_loading_association (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_polymorphic_with_primary_key_option (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2725104&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2726608&tab=buildLog) on [master@7d1324fa42732f482329a524b0166db8dd7365e6](https://github.com/cockroachdb/cockroach/commits/7d1324fa42732f482329a524b0166db8dd7365e6):

```
		--- FAIL: InheritanceTest#test_new_with_autoload_paths (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_delete_all_on_association_clears_scope (unexpected)
		--- FAIL: FinderTest#test_find_by!_with_hash_conditions_returns_the_first_matching_record (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_update_counter_caches_on_delete_with_dependent_nullify (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_many_through_polymorphic_has_manys_works (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_build_by_new_record (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_one_mapping_and_key_value_being_attribute_value (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_delete_through_belongs_to_with_dependent_nullify (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_belongs_to_with_dependent_delete_all (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_associate_with_create_bang_and_invalid_options (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_get_ids_for_has_many_through_with_conditions_should_not_preload (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_decimal_for_string_column (unexpected)
		--- FAIL: PersistenceTest#test_update_attribute_for_readonly_attribute (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_add_two_instance_and_then_deleting (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_with_escaped_characters (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_not_polymorphic_id_and_type_as_nand (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_collection_singular_ids_setter (unexpected)
		--- FAIL: FinderTest#test_third_to_last (unexpected)
		--- FAIL: FinderTest#test_find_on_hash_conditions_with_explicit_table_name_and_aggregate (unexpected)
		--- FAIL: FinderTest#test_exists_with_polymorphic_relation (unexpected)
		--- FAIL: FinderTest#test_find_one_message_on_primary_key (unexpected)
		--- FAIL: PersistenceTest#test_decrement_with_touch_updates_timestamps (unexpected)
		--- FAIL: FinderTest#test_second_with_offset (unexpected)
		--- FAIL: CockroachDB::WithAnnotationsTest#test_has_one_with_annotation_includes_a_query_comment (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_strong_parameters (unexpected)
		--- FAIL: FinderTest#test_exists_with_distinct_association_includes_and_limit (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_has_many_sti (unexpected)
		--- FAIL: FinderTest#test_hash_condition_utc_time_interpolation_with_default_timezone_local (unexpected)
		--- FAIL: PersistenceTest#test_update_attributes! (unexpected)
		--- FAIL: PersistenceTest#test_update_all_with_hash (unexpected)
		--- FAIL: FinderTest#test_third_have_primary_key_order_by_default (unexpected)
		--- FAIL: FinderTest#test_find_by_with_associations (unexpected)
		--- FAIL: FinderTest#test_exists_with_distinct_and_offset_and_joins (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_rewhere_on_root (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_singular_ids_are_reloaded_after_collection_concat (unexpected)
		--- FAIL: PersistenceTest#test_update_column_should_not_leave_the_object_dirty (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_scoped_find_on_through_association_doesnt_return_read_only_records (unexpected)
		--- FAIL: PersistenceTest#test_destroy (unexpected)
		--- FAIL: PersistenceTest#test_class_level_destroy_is_affected_by_scoping (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_deleting_from_has_many_through_a_belongs_to_should_not_try_to_update_counter (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_find_on_has_many_association_collection_with_include_and_conditions (unexpected)
		--- FAIL: FinderTest#test_exists_with_empty_table_and_no_args_given (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_clear_associations (unexpected)
		--- FAIL: FinderTest#test_exists_with_order_and_distinct (unexpected)
		--- FAIL: PersistenceTest#test_update_sti_type (unexpected)
		--- FAIL: PersistenceTest#test_save! (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2726608&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2728767&tab=buildLog) on [master@206e8178de9ac124bfde5529d8a9b858734038bb](https://github.com/cockroachdb/cockroach/commits/206e8178de9ac124bfde5529d8a9b858734038bb):

```
		--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_preloaded_records_are_not_duplicated (unexpected)
		--- FAIL: BasicsTest#test_find_reverse_ordered_last (unexpected)
		--- FAIL: FinderTest#test_joins_dont_clobber_id (unexpected)
		--- FAIL: RelationTest#test_create_with_polymorphic_association (unexpected)
		--- FAIL: BasicsTest#test_equality (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_collection_singular_ids_setter_raises_exception_when_invalid_ids_set (unexpected)
		--- FAIL: FinderTest#test_find_by_id_with_hash (unexpected)
		--- FAIL: RelationTest#test_find_all_using_where_with_relation_with_joins (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_size_of_through_association_should_increase_correctly_when_has_many_association_is_added (unexpected)
		--- FAIL: FinderTest#test_exists_with_scope (unexpected)
		--- FAIL: AttributeMethodsTest#test_setting_a_time_zone-aware_attribute_to_UTC (unexpected)
		--- FAIL: PostgresqlBitStringTest#test_roundtrip (unexpected)
		--- FAIL: ReflectionTest#test_scope_chain_does_not_interfere_with_hmt_with_polymorphic_case (unexpected)
		--- FAIL: RelationTest#test_unscope_with_merge (unexpected)
		--- FAIL: ReflectionTest#test_includes_accepts_symbols (unexpected)
		--- FAIL: CockroachDB::FixturesTest#test_clean_fixtures (unexpected)
		--- FAIL: FinderTest#test_find_on_hash_conditions_with_multiple_ranges (unexpected)
		--- FAIL: ReflectionTest#test_validate_association_if_autosave (unexpected)
		--- FAIL: RelationTest#test_finding_with_asc_order_with_string (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_two_levels_with_condition (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_validation_does_not_validate_stale_association_target (unexpected)
		--- FAIL: RelationTest#test_alias_tracker_respects_a_custom_table (unexpected)
		--- FAIL: ReflectionTest#test_active_record_primary_key_raises_when_missing_primary_key (unexpected)
		--- FAIL: JsonSerializationTest#test_should_allow_attribute_filtering_with_only (unexpected)
		--- FAIL: BasicsTest#test_records_without_an_id_have_unique_hashes (unexpected)
		--- FAIL: FinderTest#test_find_by_association_subquery (unexpected)
		--- FAIL: RelationTest#test_finding_with_assoc_reorder_by_aliased_attributes (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_replace_by_id_order_is_preserved (unexpected)
		--- FAIL: BasicsTest#test_table_name_guesses (unexpected)
		--- FAIL: FinderTest#test_last_with_integer_and_order_should_keep_the_order (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_belongs_to_sti (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_save_fails_for_invalid_belongs_to (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_destroy_association (unexpected)
		--- FAIL: RelationTest#test_create_or_find_by_with_bang_within_transaction (unexpected)
		--- FAIL: HasOneAssociationsTest#test_nullify_on_polymorphic_association (unexpected)
		--- FAIL: BasicsTest#test_ignored_columns_not_included_in_SELECT (unexpected)
		--- FAIL: HasOneAssociationsTest#test_create_association (unexpected)
		--- FAIL: HasOneAssociationsTest#test_can_marshal_has_one_association_with_nil_target (unexpected)
		--- FAIL: AttributeMethodsTest#test_define_attribute_method_works_with_both_symbol_and_string (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_delete_all_for_with_dependent_option_destroy (unexpected)
		--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyInCreateTest#test_foreign_keys_can_be_created_with_the_table (unexpected)
		--- FAIL: AttributeMethodsTest#test_attribute_predicates_respect_access_control (unexpected)
		--- FAIL: BasicsTest#test_create_after_initialize_with_block (unexpected)
		--- FAIL: RelationTest#test_to_json (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnAHasOneAssociation#test_assignment_before_either_saved (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_select_chosen_fields_only (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2728767&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2733649&tab=buildLog) on [master@9ba48738bc511ad6954682cab41e23b8492facd8](https://github.com/cockroachdb/cockroach/commits/9ba48738bc511ad6954682cab41e23b8492facd8):

```
		--- FAIL: BelongsToAssociationsTest#test_attributes_are_set_without_error_when_initialized_from_belongs_to_association_with_array_in_where_clause (unexpected)
		--- FAIL: AttributeMethodsTest#test_inherited_custom_accessors_with_reserved_names (unexpected)
		--- FAIL: EagerAssociationTest#test_preconfigured_includes_with_belongs_to (unexpected)
		--- FAIL: TestNestedAttributesOnAHasManyAssociation#test_should_be_possible_to_destroy_a_record (unexpected)
		--- FAIL: RelationTest#test_do_not_double_quote_string_id (unexpected)
		--- FAIL: HasManyAssociationsTest#test_finding_with_foreign_key (unexpected)
		--- FAIL: TransactionsWithTransactionalFixturesTest#test_no_automatic_savepoint_for_inner_transaction (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_and_eager_loading_of_optional_instance_dependent_associations_is_not_supported (unexpected)
		--- FAIL: EagerAssociationTest#test_joins_with_includes_should_preload_via_joins (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_with_bang_bubbles_errors_from_associations (unexpected)
		--- FAIL: ReflectionTest#test_has_many_through_reflection (unexpected)
		--- FAIL: RelationTest#test_dynamic_find_by_attributes_bang (unexpected)
		--- FAIL: I18nValidationTest#test_validates_uniqueness_of_on_generated_message_given_on_condition (unexpected)
		--- FAIL: ReflectionTest#test_class_for_class_name (unexpected)
		--- FAIL: ActiveRecord::StatementCacheTest#test_statement_cache_id (unexpected)
		--- FAIL: MigratorTest#test_up_calls_up (unexpected)
		--- FAIL: AttributeMethodsTest#test_custom_field_attribute_predicate (unexpected)
		--- FAIL: BasicsTest#test_preserving_time_objects_with_time_with_zone_conversion_to_default_timezone_utc (unexpected)
		--- FAIL: SuppressorTest#test_suppresses_validations_on_create (unexpected)
		--- FAIL: ActiveRecord::DupTest#test_dup_with_changes (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_with_has_one_through_an_sti_with_after_initialize (unexpected)
		--- FAIL: IntegrationTest#test_to_param_class_method_multibyte_character (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_load_has_one_quotes_table_and_column_names (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_reflect_the_most_recent_change (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_with_scope_that_has_joined_same_table_with_parent_relation (unexpected)
		--- FAIL: TimestampTest#test_touching_a_record_touches_polymorphic_record (unexpected)
		--- FAIL: BasicsTest#test_records_of_different_classes_have_different_hashes (unexpected)
		--- FAIL: EnumTest#test_assign_long_empty_string_value (unexpected)
		--- FAIL: TestAutosaveAssociationOnAHasManyAssociation#test_should_not_use_default_invalid_error_on_associated_models (unexpected)
		--- FAIL: EagerAssociationTest#test_association_loading_notification (unexpected)
		--- FAIL: RelationTest#test_select_with_subquery_in_from_does_not_use_original_table_name (unexpected)
		--- FAIL: JsonAttributeTest#test_select_array_json_value (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_symbols_as_keys (unexpected)
		--- FAIL: CalculationsTest#test_pluck_in_relation (unexpected)
		--- FAIL: CounterCacheTest#test_update_multiple_counters (unexpected)
		--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_a_marked_for_destruction_record_should_not_be_be_marked_after_reload (unexpected)
		--- FAIL: RelationTest#test_select_takes_a_variable_list_of_args (unexpected)
		--- FAIL: OptimisticLockingTest#test_polymorphic_destroy_with_dependencies_and_lock_version (unexpected)
		--- FAIL: PersistenceTest#test_update_columns_with_one_readonly_attribute (unexpected)
		--- FAIL: RelationTest#test_reverse_order_with_function_other_predicates (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnAHasOneAssociation#test_assignment_before_either_saved (unexpected)
		--- FAIL: HasOneAssociationsTest#test_build_with_block (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_belongs_to_with_dependent_nullify (unexpected)
		--- FAIL: AttributeMethodsTest#test_attribute_keys_on_a_new_instance (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dangerous_association_name_raises_ArgumentError (unexpected)
		--- FAIL: OptimisticLockingTest#test_lock_new_when_explicitly_passing_value (unexpected)
		--- FAIL: InverseHasOneTests#test_parent_instance_should_be_shared_with_child_on_find (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2733649&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2738824&tab=buildLog) on [master@6de4313ec216161c79fe725fcc31fc87ef1804ea](https://github.com/cockroachdb/cockroach/commits/6de4313ec216161c79fe725fcc31fc87ef1804ea):

```
		--- FAIL: RelationTest#test_explicit_create_with (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_association_name_is_the_same_as_join_table_name (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_find_scoped_grouped (unexpected)
		--- FAIL: BasicsTest#test_load (unexpected)
		--- FAIL: RelationTest#test_loaded_relations_cannot_be_mutated_by_extending! (unexpected)
		--- FAIL: BasicsTest#test_quote (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object_with_explicit_sti_base_class (unexpected)
		--- FAIL: HasManyAssociationsTest#test_adding_array_and_collection (unexpected)
		--- FAIL: ActiveRecord::DupTest#test_dup_record_not_persisted_after_rollback_transaction (unexpected)
		--- FAIL: TransactionTest#test_cancellation_from_before_filters_rollbacks_in_save (unexpected)
		--- FAIL: PersistenceTest#test_delete_many (unexpected)
		--- FAIL: HasManyAssociationsTest#test_attributes_are_set_when_initialized_from_polymorphic_has_many_null_relationship (unexpected)
		--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_scope_array (unexpected)
		--- FAIL: FinderTest#test_find_with_entire_select_statement (unexpected)
		--- FAIL: BasicsTest#test_dup (unexpected)
		--- FAIL: BasicsTest#test_has_attribute_with_symbol (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_updating_attributes_on_non_rich_associations (unexpected)
		--- FAIL: BasicsTest#test_deleting_a_record_raises_if_preventing_writes (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all (unexpected)
		--- FAIL: CalculationsTest#test_pluck_with_multiple_columns_and_includes (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_find_with_merged_options (unexpected)
		--- FAIL: CounterCacheTest#test_reset_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_replace_with_new (unexpected)
		--- FAIL: EagerAssociationTest#test_preload_has_many_using_primary_key (unexpected)
		--- FAIL: EagerAssociationTest#test_loading_polymorphic_association_with_mixed_table_conditions (unexpected)
		--- FAIL: PersistenceTest#test_update (unexpected)
		--- FAIL: HasOneAssociationsTest#test_building_the_associated_object_with_an_invalid_type (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnAHasManyAssociation#test_assign_ids (unexpected)
		--- FAIL: BasicsTest#test_equality_with_blank_ids (unexpected)
		--- FAIL: HasOneAssociationsTest#test_association_change_calls_destroy (unexpected)
		--- FAIL: NamedScopingTest#test_chaining_applies_last_conditions_when_creating (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_with_a_polymorphic_association_won't_work_consistently (unexpected)
		--- FAIL: RelationTest#test_scoped_build (unexpected)
		--- FAIL: RelationTest#test_delete_by (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_scoped_grouped_having (unexpected)
		--- FAIL: CounterCacheTest#test_update_multiple_counters_with_touch:_:written_on (unexpected)
		--- FAIL: PersistenceTest#test_save_for_record_with_only_primary_key_that_is_provided (unexpected)
		--- FAIL: HasManyAssociationsTest#test_clearing_a_dependent_association_collection (unexpected)
		--- FAIL: ActiveRecord::InvertibleMigrationTest#test_up_only (unexpected)
		--- FAIL: FinderTest#test_find_with_ids_and_limit (unexpected)
		--- FAIL: BasicsTest#test_toggle_attribute (unexpected)
		--- FAIL: RelationTest#test_finding_with_subquery (unexpected)
		--- FAIL: RelationTest#test_finding_with_assoc_order_by_aliased_attributes (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_attributes_are_being_set_when_initialized_from_belongs_to_association_with_where_clause (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_assign_ids (unexpected)
		--- FAIL: BasicsTest#test_set_table_name_symbol_converted_to_string (unexpected)
		--- FAIL: FinderTest#test_condition_local_time_interpolation_with_default_timezone_utc (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2738824&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2747304&tab=buildLog) on [master@b703e663da8ededaee2e28fc39a24e3880ae54cf](https://github.com/cockroachdb/cockroach/commits/b703e663da8ededaee2e28fc39a24e3880ae54cf):

```
		--- FAIL: PersistenceTest#test_becomes (unexpected)
		--- FAIL: HasManyAssociationsTest#test_create_without_loading_association (unexpected)
		--- FAIL: CalculationsTest#test_minimum_with_from_option (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_join_for_count (unexpected)
		--- FAIL: CalculationsTest#test_count_selected_arel_attribute (unexpected)
		--- FAIL: HasManyAssociationsTest#test_in_memory_replacement_executes_no_queries (unexpected)
		--- FAIL: PersistenceTest#test_update_all (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_two_levels_and_one_level (unexpected)
		--- FAIL: CalculationsTest#test_count_takes_attribute_type_precedence_over_database_type (unexpected)
		--- FAIL: HasManyAssociationsTest#test_sti_subselect_count (unexpected)
		--- FAIL: PersistenceTest#test_update_column_with_model_having_primary_key_other_than_id (unexpected)
		--- FAIL: CalculationsTest#test_#skip_query_cache!_for_a_simple_calculation (unexpected)
		--- FAIL: PersistenceTest#test_delete_new_record (unexpected)
		--- FAIL: HasManyAssociationsTest#test_included_in_collection (unexpected)
		--- FAIL: HasManyAssociationsTest#test_collection_not_empty_after_building (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_cascaded_eager_association_loading_with_duplicated_includes (unexpected)
		--- FAIL: PersistenceTest#test_becomes_errors_base (unexpected)
		--- FAIL: CalculationsTest#test_aggregate_attribute_on_custom_type (unexpected)
		--- FAIL: HasManyAssociationsTest#test_finding (unexpected)
		--- FAIL: CalculationsTest#test_pluck_loaded_relation_multiple_columns (unexpected)
		--- FAIL: HasManyAssociationsTest#test_counting_with_association_limit (unexpected)
		--- FAIL: PersistenceTest#test_increment_updates_counter_in_db_using_offset (unexpected)
		--- FAIL: CalculationsTest#test_should_not_use_alias_for_grouped_field (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_on_association_with_relation_performs_subselect_not_two_queries (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_integer_for_binary_column (unexpected)
		--- FAIL: CalculationsTest#test_should_group_by_scoped_field (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dynamic_find_should_respect_association_order_for_through (unexpected)
		--- FAIL: HasManyAssociationsTest#test_defining_has_many_association_with_delete_all_dependency_lazily_evaluates_target_class (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_empty_with_counter_cache (unexpected)
		--- FAIL: CalculationsTest#test_count_with_from_option (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_cascaded_eager_association_loading_with_join_for_count (unexpected)
		--- FAIL: ModulesTest#test_assign_ids (unexpected)
		--- FAIL: CalculationsTest#test_distinct_count_with_order_and_offset (unexpected)
		--- FAIL: CalculationsTest#test_select_avg_with_joins_and_group_by_as_virtual_attribute_with_ar (unexpected)
		--- FAIL: CalculationsTest#test_#skip_query_cache!_for_#pluck (unexpected)
		--- FAIL: CalculationsTest#test_should_group_by_multiple_fields (unexpected)
		--- FAIL: HasManyAssociationsTest#test_merging_with_custom_attribute_writer (unexpected)
		--- FAIL: AssociationsTest#test_force_reload (unexpected)
		--- FAIL: HasManyAssociationsTest#test_first_or_create!_adds_the_record_to_the_association (unexpected)
		--- FAIL: PersistenceTest#test_update_column_should_not_modify_updated_at (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_none_should_return_false_if_any (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_size_on_an_association_that_has_not_been_loaded_performs_a_query (unexpected)
		--- FAIL: PersistenceTest#test_persisted_returns_boolean (unexpected)
		--- FAIL: CalculationsTest#test_should_return_decimal_average_of_integer_field (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_has_many_sti (unexpected)
		--- FAIL: HasManyAssociationsTest#test_to_a_should_dup_target (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2747304&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2751365&tab=buildLog) on [master@15a185606d5e80b47d9fdd0ed4f54cfe29c527c6](https://github.com/cockroachdb/cockroach/commits/15a185606d5e80b47d9fdd0ed4f54cfe29c527c6):

```
		--- FAIL: EagerAssociationTest#test_preloading_with_has_one_through_an_sti_with_after_initialize (unexpected)
		--- FAIL: YamlSerializationTest#test_new_records_remain_new_after_round_trip (unexpected)
		--- FAIL: CalculationsTest#test_sum_with_block_and_column_name_raises_an_error (unexpected)
		--- FAIL: AttributeMethodsTest#test_write_attribute_can_write_aliased_attributes_as_well (unexpected)
		--- FAIL: HasManyAssociationsTest#test_include_checks_if_record_exists_if_target_not_loaded (unexpected)
		--- FAIL: BasicsTest#test_dup (unexpected)
		--- FAIL: FinderTest#test_second (unexpected)
		--- FAIL: YamlSerializationTest#test_psych_roundtrip_new_object (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_with_one_condition_being_aggregate_and_another_not (unexpected)
		--- FAIL: HasManyAssociationsTest#test_adding_a_collection (unexpected)
		--- FAIL: PersistenceTest#test_update_attribute_with_one_updated (unexpected)
		--- FAIL: InheritanceTest#test_alt_eager_loading (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnAHasOneAssociation#test_callbacks_firing_order_on_update (unexpected)
		--- FAIL: ReflectionTest#test_association_reflection_in_modules (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_loading_with_conditions_on_string_joined_table_preloads (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_association_assignment_sticks (unexpected)
		--- FAIL: RelationTest#test_raising_exception_on_invalid_hash_params (unexpected)
		--- FAIL: HasManyAssociationsTest#test_build_the_association_with_an_array (unexpected)
		--- FAIL: FinderTest#test_exists_passing_active_record_object_is_not_permitted (unexpected)
		--- FAIL: InheritanceTest#test_where_new_with_unrelated_type (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_replace_counter_cache (unexpected)
		--- FAIL: PersistenceTest#test_update_for_record_with_only_primary_key (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_and_removed_parent (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_where_on_polymorphic_association_with_empty_array (unexpected)
		--- FAIL: HasOneAssociationsTest#test_build (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_by_integer_id (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter_with_reassigning (unexpected)
		--- FAIL: BasicsTest#test_values_at_accepts_array_argument (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnAHasOneAssociation#test_should_save_parent_but_not_invalid_child (unexpected)
		--- FAIL: HasManyAssociationsTest#test_clearing_updates_counter_cache_when_inverse_counter_cache_is_a_symbol_with_dependent_destroy (unexpected)
		--- FAIL: BasicsTest#test_limit_without_comma (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_readonly_association (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_with_conditions_join_model_with_include (unexpected)
		--- FAIL: CalculationsTest#test_distinct_joins_count_with_order_and_limit (unexpected)
		--- FAIL: SignedIdTest#test_find_signed_record_with_a_bang (unexpected)
		--- FAIL: InheritanceTest#test_class_without_store_full_sti_class_returns_demodulized_name (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_scope_can_unscope (unexpected)
		--- FAIL: RelationTest#test_size_with_eager_loading_and_custom_select_and_order (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_limit_and_conditions (unexpected)
		--- FAIL: CalculationsTest#test_pluck_and_distinct (unexpected)
		--- FAIL: ReflectionTest#test_always_validate_association_if_explicit (unexpected)
		--- FAIL: ReflectionTest#test_association_primary_key_raises_when_missing_primary_key (unexpected)
		--- FAIL: FinderTest#test_find_by_records (unexpected)
		--- FAIL: UniquenessValidationTest#test_validate_case_sensitive_uniqueness_with_special_sql_like_chars (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_with_a_polymorphic_association_and_using_the_existential_predicate (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_through_scope_is_affected_by_unscoping (unexpected)
		--- FAIL: DateTimeTest#test_assign_empty_date (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList21_1) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2751365&tab=artifacts#/activerecord)
Related:
- #58992 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.2)

- #53730 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/ccl/importccl/import_stmt_test.go
+++ b/pkg/ccl/importccl/import_stmt_test.go
@@ -5668,7 +5668,7 @@ func TestImportPgDump(t *testing.T) {
 			if c.expected == expectAll {
 				sqlDB.CheckQueryResults(t, `SHOW CREATE TABLE seqtable`, [][]string{{
 					"seqtable", `CREATE TABLE public.seqtable (
-	a INT8 NULL DEFAULT nextval('foo.public.a_seq':::STRING::REGCLASS),
+	a INT8 NULL DEFAULT nextval('public.a_seq':::STRING::REGCLASS),
 	b INT8 NULL,
 	rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
 	CONSTRAINT "primary" PRIMARY KEY (rowid ASC),

--- a/pkg/sql/catalog/schemaexpr/column.go
+++ b/pkg/sql/catalog/schemaexpr/column.go
@@ -302,11 +302,19 @@ func replaceIDsWithFQNames(
 		if err != nil {
 			return true, expr, nil //nolint:returnerrcheck
 		}
+
+		// Omit the database qualification if the sequence lives in the current database.
+		currDb := semaCtx.TableNameResolver.CurrentDatabase()
+		if seqName.Catalog() == currDb {
+			seqName.CatalogName = ""
+			seqName.ExplicitCatalog = false
+		}
+
 		// Swap out this node to use the qualified table name for the sequence.
 		return false, &tree.CastExpr{
 			Type:       types.RegClass,
 			SyntaxMode: tree.CastShort,
-			Expr:       tree.NewStrVal(seqName.FQString()),
+			Expr:       tree.NewStrVal(seqName.String()),
 		}, nil
 	}
 

--- a/pkg/sql/logictest/testdata/logic_test/drop_sequence
+++ b/pkg/sql/logictest/testdata/logic_test/drop_sequence
@@ -31,7 +31,7 @@ query TT
 SHOW CREATE TABLE t1
 ----
 t1  CREATE TABLE public.t1 (
-    i INT8 NOT NULL DEFAULT nextval('test.public.drop_test':::STRING::REGCLASS),
+    i INT8 NOT NULL DEFAULT nextval('public.drop_test':::STRING::REGCLASS),
     rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
     CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
     FAMILY "primary" (i, rowid)
@@ -93,7 +93,7 @@ SHOW CREATE TABLE foo
 ----
 foo  CREATE TABLE public.foo (
      i INT8 NOT NULL DEFAULT nextval('other_db.public.s':::STRING::REGCLASS),
-     j INT8 NOT NULL DEFAULT nextval('test.public.s':::STRING::REGCLASS),
+     j INT8 NOT NULL DEFAULT nextval('public.s':::STRING::REGCLASS),
      rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
      CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
      FAMILY fam_0_i_j_rowid (i, j, rowid)
@@ -115,7 +115,7 @@ SHOW CREATE TABLE foo
 ----
 foo  CREATE TABLE public.foo (
      i INT8 NOT NULL,
-     j INT8 NOT NULL DEFAULT nextval('test.public.s':::STRING::REGCLASS),
+     j INT8 NOT NULL DEFAULT nextval('public.s':::STRING::REGCLASS),
      rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
      CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
      FAMILY fam_0_i_j_rowid (i, j, rowid)
@@ -151,8 +151,8 @@ query TT
 SHOW CREATE TABLE bar
 ----
 bar  CREATE TABLE public.bar (
-     i INT8 NOT NULL DEFAULT nextval('test.other_sc.s':::STRING::REGCLASS),
-     j INT8 NOT NULL DEFAULT nextval('test.public.s':::STRING::REGCLASS),
+     i INT8 NOT NULL DEFAULT nextval('other_sc.s':::STRING::REGCLASS),
+     j INT8 NOT NULL DEFAULT nextval('public.s':::STRING::REGCLASS),
      rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
      CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
      FAMILY fam_0_i_j_rowid (i, j, rowid)
@@ -174,7 +174,7 @@ SHOW CREATE TABLE bar
 ----
 bar  CREATE TABLE public.bar (
      i INT8 NOT NULL,
-     j INT8 NOT NULL DEFAULT nextval('test.public.s':::STRING::REGCLASS),
+     j INT8 NOT NULL DEFAULT nextval('public.s':::STRING::REGCLASS),
      rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
      CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
      FAMILY fam_0_i_j_rowid (i, j, rowid)

--- a/pkg/sql/logictest/testdata/logic_test/int_size
+++ b/pkg/sql/logictest/testdata/logic_test/int_size
@@ -134,7 +134,7 @@ query TT
 SHOW CREATE TABLE i4_sql_sequence
 ----
 i4_sql_sequence  CREATE TABLE public.i4_sql_sequence (
-                 a INT4 NOT NULL DEFAULT nextval('test.public.i4_sql_sequence_a_seq':::STRING::REGCLASS),
+                 a INT4 NOT NULL DEFAULT nextval('public.i4_sql_sequence_a_seq':::STRING::REGCLASS),
                  rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
                  CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
                  FAMILY "primary" (a, rowid)
@@ -150,7 +150,7 @@ query TT
 SHOW CREATE TABLE i8_sql_sequence
 ----
 i8_sql_sequence  CREATE TABLE public.i8_sql_sequence (
-                 a INT8 NOT NULL DEFAULT nextval('test.public.i8_sql_sequence_a_seq':::STRING::REGCLASS),
+                 a INT8 NOT NULL DEFAULT nextval('public.i8_sql_sequence_a_seq':::STRING::REGCLASS),
                  rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
                  CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
                  FAMILY "primary" (a, rowid)
@@ -170,7 +170,7 @@ query TT
 SHOW CREATE TABLE i4_virtual_sequence
 ----
 i4_virtual_sequence  CREATE TABLE public.i4_virtual_sequence (
-                     a INT8 NOT NULL DEFAULT nextval('test.public.i4_virtual_sequence_a_seq':::STRING::REGCLASS),
+                     a INT8 NOT NULL DEFAULT nextval('public.i4_virtual_sequence_a_seq':::STRING::REGCLASS),
                      rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
                      CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
                      FAMILY "primary" (a, rowid)
@@ -186,7 +186,7 @@ query TT
 SHOW CREATE TABLE i8_virtual_sequence
 ----
 i8_virtual_sequence  CREATE TABLE public.i8_virtual_sequence (
-                     a INT8 NOT NULL DEFAULT nextval('test.public.i8_virtual_sequence_a_seq':::STRING::REGCLASS),
+                     a INT8 NOT NULL DEFAULT nextval('public.i8_virtual_sequence_a_seq':::STRING::REGCLASS),
                      rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
                      CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
                      FAMILY "primary" (a, rowid)

--- a/pkg/sql/logictest/testdata/logic_test/sequences_regclass
+++ b/pkg/sql/logictest/testdata/logic_test/sequences_regclass
@@ -8,13 +8,13 @@ statement ok
 CREATE SEQUENCE test_seq
 
 statement ok
-CREATE SEQUENCE test_seq2
+CREATE DATABASE diff_db
+
+statement ok
+CREATE SEQUENCE diff_db.test_seq
 
 let $test_seq_id
 SELECT 'test_seq'::regclass::int
-
-let $test_seq2_id
-SELECT 'test_seq2'::regclass::int
 
 statement ok
 CREATE TABLE foo (i SERIAL PRIMARY KEY)
@@ -29,19 +29,19 @@ statement ok
 ALTER TABLE foo ADD COLUMN l INT NOT NULL
 
 statement ok
-ALTER TABLE FOO ALTER COLUMN l SET DEFAULT currval('test_seq2')
+ALTER TABLE FOO ALTER COLUMN l SET DEFAULT currval('diff_db.test_seq')
 
 statement ok
-SELECT nextval('test_seq2')
+SELECT nextval('diff_db.test_seq')
 
 query TT
 SHOW CREATE TABLE foo
 ----
 foo  CREATE TABLE public.foo (
-     i INT8 NOT NULL DEFAULT nextval('test.public.foo_i_seq':::STRING::REGCLASS),
-     j INT8 NOT NULL DEFAULT nextval('test.public.test_seq':::STRING::REGCLASS),
-     k INT8 NOT NULL DEFAULT nextval('test.public.foo_k_seq':::STRING::REGCLASS),
-     l INT8 NOT NULL DEFAULT currval('test.public.test_seq2':::STRING::REGCLASS),
+     i INT8 NOT NULL DEFAULT nextval('public.foo_i_seq':::STRING::REGCLASS),
+     j INT8 NOT NULL DEFAULT nextval('public.test_seq':::STRING::REGCLASS),
+     k INT8 NOT NULL DEFAULT nextval('public.foo_k_seq':::STRING::REGCLASS),
+     l INT8 NOT NULL DEFAULT currval('diff_db.public.test_seq':::STRING::REGCLASS),
      CONSTRAINT "primary" PRIMARY KEY (i ASC),
      FAMILY "primary" (i, j, k, l)
 )
@@ -130,9 +130,9 @@ query TT
 SHOW CREATE TABLE bar
 ----
 bar  CREATE TABLE public.bar (
-     i INT8 NOT NULL DEFAULT nextval('test.public.new_bar_i_seq':::STRING::REGCLASS),
-     j INT8 NOT NULL DEFAULT currval('test.public.new_s1':::STRING::REGCLASS),
-     k INT8 NOT NULL DEFAULT nextval('test.public.new_s2':::STRING::REGCLASS),
+     i INT8 NOT NULL DEFAULT nextval('public.new_bar_i_seq':::STRING::REGCLASS),
+     j INT8 NOT NULL DEFAULT currval('public.new_s1':::STRING::REGCLASS),
+     k INT8 NOT NULL DEFAULT nextval('public.new_s2':::STRING::REGCLASS),
      CONSTRAINT "primary" PRIMARY KEY (i ASC),
      FAMILY fam_0_i_j_k (i, j, k)
 )
@@ -251,9 +251,9 @@ query TT
 SHOW CREATE TABLE tb
 ----
 tb  CREATE TABLE public.tb (
-    i INT8 NOT NULL DEFAULT nextval('test.test_schema.tb_i_seq':::STRING::REGCLASS),
-    j INT8 NOT NULL DEFAULT nextval('test.test_schema.sc_s1':::STRING::REGCLASS),
-    k INT8 NOT NULL DEFAULT currval('test.test_schema.sc_s2':::STRING::REGCLASS),
+    i INT8 NOT NULL DEFAULT nextval('test_schema.tb_i_seq':::STRING::REGCLASS),
+    j INT8 NOT NULL DEFAULT nextval('test_schema.sc_s1':::STRING::REGCLASS),
+    k INT8 NOT NULL DEFAULT currval('test_schema.sc_s2':::STRING::REGCLASS),
     CONSTRAINT "primary" PRIMARY KEY (i ASC),
     FAMILY fam_0_i_j_k (i, j, k)
 )
@@ -306,9 +306,9 @@ query TT
 SHOW CREATE TABLE new_test_schema.foo
 ----
 new_test_schema.foo  CREATE TABLE new_test_schema.foo (
-                     i INT8 NOT NULL DEFAULT nextval('test.new_test_schema.foo_i_seq':::STRING::REGCLASS),
-                     j INT8 NOT NULL DEFAULT nextval('test.new_test_schema.s3':::STRING::REGCLASS),
-                     k INT8 NOT NULL DEFAULT currval('test.new_test_schema.s4':::STRING::REGCLASS),
+                     i INT8 NOT NULL DEFAULT nextval('new_test_schema.foo_i_seq':::STRING::REGCLASS),
+                     j INT8 NOT NULL DEFAULT nextval('new_test_schema.s3':::STRING::REGCLASS),
+                     k INT8 NOT NULL DEFAULT currval('new_test_schema.s4':::STRING::REGCLASS),
                      CONSTRAINT "primary" PRIMARY KEY (i ASC),
                      FAMILY fam_0_i_j_k (i, j, k)
 )

--- a/pkg/sql/logictest/testdata/logic_test/serial
+++ b/pkg/sql/logictest/testdata/logic_test/serial
@@ -117,9 +117,9 @@ query TT
 SHOW CREATE TABLE serial
 ----
 serial  CREATE TABLE public.serial (
-        a INT8 NOT NULL DEFAULT nextval('test.public.serial_a_seq':::STRING::REGCLASS),
+        a INT8 NOT NULL DEFAULT nextval('public.serial_a_seq':::STRING::REGCLASS),
         b INT8 NULL DEFAULT 7:::INT8,
-        c INT8 NOT NULL DEFAULT nextval('test.public.serial_c_seq2':::STRING::REGCLASS),
+        c INT8 NOT NULL DEFAULT nextval('public.serial_c_seq2':::STRING::REGCLASS),
         CONSTRAINT "primary" PRIMARY KEY (a ASC),
         UNIQUE INDEX serial_c_key (c ASC),
         FAMILY "primary" (a, b, c)
@@ -160,8 +160,8 @@ query TT
 SHOW CREATE TABLE smallbig
 ----
 smallbig  CREATE TABLE public.smallbig (
-          a INT8 NOT NULL DEFAULT nextval('test.public.smallbig_a_seq':::STRING::REGCLASS),
-          b INT8 NOT NULL DEFAULT nextval('test.public.smallbig_b_seq':::STRING::REGCLASS),
+          a INT8 NOT NULL DEFAULT nextval('public.smallbig_a_seq':::STRING::REGCLASS),
+          b INT8 NOT NULL DEFAULT nextval('public.smallbig_b_seq':::STRING::REGCLASS),
           c INT8 NULL,
           rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
           CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
@@ -183,9 +183,9 @@ query TT
 SHOW CREATE TABLE serials
 ----
 serials  CREATE TABLE public.serials (
-         a INT8 NOT NULL DEFAULT nextval('test.public.serials_a_seq':::STRING::REGCLASS),
-         b INT8 NOT NULL DEFAULT nextval('test.public.serials_b_seq':::STRING::REGCLASS),
-         c INT8 NOT NULL DEFAULT nextval('test.public.serials_c_seq':::STRING::REGCLASS),
+         a INT8 NOT NULL DEFAULT nextval('public.serials_a_seq':::STRING::REGCLASS),
+         b INT8 NOT NULL DEFAULT nextval('public.serials_b_seq':::STRING::REGCLASS),
+         c INT8 NOT NULL DEFAULT nextval('public.serials_c_seq':::STRING::REGCLASS),
          d INT8 NULL,
          rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
          CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
@@ -222,9 +222,9 @@ query TT
 SHOW CREATE TABLE serial
 ----
 serial  CREATE TABLE public.serial (
-        a INT8 NOT NULL DEFAULT nextval('test.public.serial_a_seq1':::STRING::REGCLASS),
+        a INT8 NOT NULL DEFAULT nextval('public.serial_a_seq1':::STRING::REGCLASS),
         b INT8 NULL DEFAULT 7:::INT8,
-        c INT8 NOT NULL DEFAULT nextval('test.public.serial_c_seq3':::STRING::REGCLASS),
+        c INT8 NOT NULL DEFAULT nextval('public.serial_c_seq3':::STRING::REGCLASS),
         CONSTRAINT "primary" PRIMARY KEY (a ASC),
         UNIQUE INDEX serial_c_key (c ASC),
         FAMILY "primary" (a, b, c)
@@ -278,9 +278,9 @@ query TT
 SHOW CREATE TABLE "serial_MixedCase"
 ----
 "serial_MixedCase"  CREATE TABLE public."serial_MixedCase" (
-                    a INT8 NOT NULL DEFAULT nextval('test.public."serial_MixedCase_a_seq"':::STRING::REGCLASS),
+                    a INT8 NOT NULL DEFAULT nextval('public."serial_MixedCase_a_seq"':::STRING::REGCLASS),
                     b INT8 NULL DEFAULT 7:::INT8,
-                    c INT8 NOT NULL DEFAULT nextval('test.public."serial_MixedCase_c_seq"':::STRING::REGCLASS),
+                    c INT8 NOT NULL DEFAULT nextval('public."serial_MixedCase_c_seq"':::STRING::REGCLASS),
                     CONSTRAINT "primary" PRIMARY KEY (a ASC),
                     UNIQUE INDEX "serial_MixedCase_c_key" (c ASC),
                     FAMILY "primary" (a, b, c)
@@ -305,8 +305,8 @@ query TT
 SHOW CREATE TABLE smallbig
 ----
 smallbig  CREATE TABLE public.smallbig (
-          a INT2 NOT NULL DEFAULT nextval('test.public.smallbig_a_seq1':::STRING::REGCLASS),
-          b INT8 NOT NULL DEFAULT nextval('test.public.smallbig_b_seq1':::STRING::REGCLASS),
+          a INT2 NOT NULL DEFAULT nextval('public.smallbig_a_seq1':::STRING::REGCLASS),
+          b INT8 NOT NULL DEFAULT nextval('public.smallbig_b_seq1':::STRING::REGCLASS),
           c INT8 NULL,
           rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
           CONSTRAINT "primary" PRIMARY KEY (rowid ASC),
@@ -328,9 +328,9 @@ query TT
 SHOW CREATE TABLE serials
 ----
 serials  CREATE TABLE public.serials (
-         a INT2 NOT NULL DEFAULT nextval('test.public.serials_a_seq1':::STRING::REGCLASS),
-         b INT4 NOT NULL DEFAULT nextval('test.public.serials_b_seq1':::STRING::REGCLASS),
-         c INT8 NOT NULL DEFAULT nextval('test.public.serials_c_seq1':::STRING::REGCLASS),
+         a INT2 NOT NULL DEFAULT nextval('public.serials_a_seq1':::STRING::REGCLASS),
+         b INT4 NOT NULL DEFAULT nextval('public.serials_b_seq1':::STRING::REGCLASS),
+         c INT8 NOT NULL DEFAULT nextval('public.serials_c_seq1':::STRING::REGCLASS),
          d INT8 NULL,
          rowid INT8 NOT VISIBLE NOT NULL DEFAULT unique_rowid(),
          CONSTRAINT "primary" PRIMARY KEY (rowid ASC),

--- a/pkg/sql/logictest/testdata/logic_test/show_create_all_tables
+++ b/pkg/sql/logictest/testdata/logic_test/show_create_all_tables
@@ -208,7 +208,7 @@ CREATE TABLE public.c (
 );
 CREATE SEQUENCE public.s MINVALUE 1 MAXVALUE 9223372036854775807 INCREMENT 1 START 1;
 CREATE TABLE public.s_tbl (
-    id INT8 NOT NULL DEFAULT nextval('test_fk_order.public.s':::STRING::REGCLASS),
+    id INT8 NOT NULL DEFAULT nextval('public.s':::STRING::REGCLASS),
     v INT8 NULL,
     CONSTRAINT "primary" PRIMARY KEY (id ASC),
     FAMILY f1 (id, v)

--- a/pkg/sql/logictest/testdata/logic_test/show_create_all_tables_builtin
+++ b/pkg/sql/logictest/testdata/logic_test/show_create_all_tables_builtin
@@ -202,7 +202,7 @@ CREATE TABLE public.c (
 );
 CREATE SEQUENCE public.s MINVALUE 1 MAXVALUE 9223372036854775807 INCREMENT 1 START 1;
 CREATE TABLE public.s_tbl (
-    id INT8 NOT NULL DEFAULT nextval('test_fk_order.public.s':::STRING::REGCLASS),
+    id INT8 NOT NULL DEFAULT nextval('public.s':::STRING::REGCLASS),
     v INT8 NULL,
     CONSTRAINT "primary" PRIMARY KEY (id ASC),
     FAMILY f1 (id, v)

--- a/pkg/sql/sem/tree/name_resolution.go
+++ b/pkg/sql/sem/tree/name_resolution.go
@@ -262,9 +262,12 @@ type ObjectNameExistingResolver interface {
 }
 
 // QualifiedNameResolver is the helper interface to resolve qualified
-// table names given an ID and the required table kind.
+// table names given an ID and the required table kind, as well as the
+// current database to determine whether or not to include the
+// database in the qualification.
 type QualifiedNameResolver interface {
 	GetQualifiedTableNameByID(ctx context.Context, id int64, requiredType RequiredTableKind) (*TableName, error)
+	CurrentDatabase() string
 }
 
 // NameResolutionResult is an opaque reference returned by LookupObject().

